db/external_sst_file_ingestion_job.cc

```diff
@@ -317,32 +323,17 @@ Status ExternalSstFileIngestionJob::Prepare(
     }
   }
 
-  if (!status.ok()) {
-    // We failed, remove all files that we copied into the db
-    DeleteInternalFiles();
-  }
-
   return status;
 }
 
 Status ExternalSstFileIngestionJob::NeedsFlush(bool* flush_needed,
                                                SuperVersion* super_version) {
   size_t n = files_to_ingest_.size();
   autovector<UserKeyRange> ranges;
-  std::vector<std::string> keys;
   ranges.reserve(n);
-  keys.reserve(2 * n);
-  size_t ts_sz = cfd_->user_comparator()->timestamp_size();
-  // Check all ranges [begin, end] inclusively.
   for (const IngestedFileInfo& file_to_ingest : files_to_ingest_) {
-    Slice start_ukey = file_to_ingest.smallest_internal_key.user_key();
-    Slice end_ukey = file_to_ingest.largest_internal_key.user_key();
-    auto [start, end] = MaybeAddTimestampsToRange(
```

Comment:
Was this code never needed  or what?

Reply:
Yes, this path previously was never actually used. It was added kind of for preparation to bulk loading external sst files with timestamps.